ci/check_line_lengths.sh

```diff
@@ -10,7 +10,7 @@ if [ "$MAX_LINE_LENGTH" == "" ]; then
 fi
 
 if [ "$1" == "" ]; then
-  files=( src/**/*.md )
+  files=( src/**.md )
```

Comment:
Wow, I didn't realize `src/**.md` worked! Learned a new thing about globs :)

Reply:
err actually I think this change is wrong:
```
> ls src/**.md | grep building | wc -l
0
```
> ls src/**/*.md | grep building | wc -l
8
```
The actual issue is that the script is being run with `sh`, not `bash`, which doesn't support double asterisks in globs. I'll fix it.